src/dns/coredns.rs

```diff
@@ -238,8 +239,51 @@ impl CoreDns {
+                                                        match response_ip {
+                                                            IpAddr::V4(ipv4) => {
+                                                                req.add_answer(
+                                                                     Record::new()
+                                                                        .set_name(record_name.clone())
+                                                                        .set_ttl(86400)
+                                                                        .set_rr_type(RecordType::A)
+                                                                        .set_dns_class(DNSClass::IN)
+                                                                        .set_rdata(RData::A(ipv4))
+                                                                        .clone(),
+                                                                );
+                                                            },
+                                                            IpAddr::V6(ipv6) => {
+                                                                req.add_answer(
+                                                                     Record::new()
+                                                                        .set_name(record_name.clone())
+                                                                        .set_ttl(86400)
+                                                                        .set_rr_type(RecordType::AAAA)
+                                                                        .set_dns_class(DNSClass::IN)
+                                                                        .set_rdata(RData::AAAA(ipv6))
+                                                                        .clone(),
+                                                                );
+                                                            }
+                                                        }
+                                                }
+                                                reply(sender.clone(), src_address, &req);
+                                                did_reply = true;
+
+                                            }
+                                        }
+
+                                        // using default from hosts's /etc/resolv.conf failed
+                                        // as a last resort try using global configured forwarder
+                                        // default global forwarder is 1.1.1.1
+                                        if !did_reply {
+                                            if let Some(resp) = forward_dns_req(client, req.clone()).await {
+                                                reply(sender.clone(), src_address, &resp).unwrap();
+                                            }
```

Comment:
We should not fallback to a hardcoded resolver IMO. If /etc/resolv.conf does not work than this is a user error.

Reply:
agree